pkg/sql/logictest/logic_runner_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 The Cockroach Authors.
```

Comment:
I wanted to name this `logic_test.go`, but then it doesn't give you a nice "moved + a few changes" in the diff for the original `logic_test.go`. We can rename it later.